README.md

```diff
@@ -82,17 +83,25 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
-
+< valid_sentence > ::= "The"< noun_or_adjNoun >< verb_verbAdv > \
+< noun_or_adjNoun > ::= < noun >|< adj >< noun > \
+< noun > ::= "dog"|"cat" \
+< adj > ::= "smelly"|"lazy" \
+< verb_verbAdv > ::= < verb >|< verb >< adv > \
+< verb > ::= "ran"|"ate" \
+< adv > ::= "slowly"|"noisily" 
```

Comment:
This grammar doesn't allow for repeating adjectives. This structure, where you combine the noud and adjective, makes that harder to implement. I was looking for something like

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
  <noun> ::= "dog" | "cat"
  <verb> ::= "ate" | "ran"
  <adverb> ::= "slowly" | "noisily" | ""
~~~

# -3